doc/manuals/orion-api.md

```diff
@@ -3198,17 +3198,17 @@ A `subject` contains the following subfields:
 
 | Parameter                                    | Optional | Type   | Description                                                                     |
 |----------------------------------------------|----------|--------|---------------------------------------------------------------------------------|
-| `entities`                                   | ✓        | string | A list of objects, each one composed of the following subfields: <ul><li><code>id</code> or <code>idPattern</code> Id or pattern of the affected entities. Both cannot be used at the same time, but one of them must be present.</li> <li><code>type</code> or <code>typePattern</code> Type or type pattern of the affected entities. Both cannot be used at the same time. If omitted, it means "any entity type".</li></ul> |
-| [`condition`](#subscriptionsubjectcondition) | ✓        | object | Condition to trigger notifications. If omitted, it means "any attribute change will trigger condition" |
+| `entities`                                   | ✓        | string | A list of objects, each one composed of the following subfields: <ul><li><code>id</code> or <code>idPattern</code> id or pattern of the affected entities. Both cannot be used at the same time, but one of them must be present.</li> <li><code>type</code> or <code>typePattern</code> Type or type pattern of the affected entities. Both cannot be used at the same time. If omitted, it means "any entity type".</li></ul> These fields should fllow the restrictions for IDs and types, be a valid regex (if pattern), and be not empty. |
```

Comment:
```suggestion
| `entities`                                   | ✓        | string | A list of objects, each one composed of the following subfields: <ul><li><code>id</code> or <code>idPattern</code> id or pattern of the affected entities. Both cannot be used at the same time, but one of them must be present.</li> <li><code>type</code> or <code>typePattern</code> Type or type pattern of the affected entities. Both cannot be used at the same time. If omitted, it means "any entity type".</li></ul> These fields should follow the restrictions for IDs and types, be a valid regex (if pattern), and be not empty. |
```

In additon, add a link to the section about identifiers restrictions.

Reply:
Done in 6e68e0f9f2e7b03701a6e5864b1026f35fa93991